pkg/model/model.go

```diff
@@ -1,260 +1,86 @@
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
+model package provides NodePackedPower, PodPowerRatio, and PodDynamicPower
+- NodePackedPower: node-level power in core, uncore, dram, and package
+- PodPowerRatio: power that each pod or system should account for when considering usage shares (ratio)
+- PodDynamicPower: power that each pod should account for when considering only its usage values
 */
 
 package model
 
 import (
-	"context"
-	"encoding/csv"
-	"encoding/json"
-	"errors"
-	"fmt"
-	"io"
-	"math"
-	"net/http"
-	"os"
-	"time"
-
 	"github.com/sustainable-computing-io/kepler/pkg/power/rapl/source"
-
-	"github.com/jszwec/csvutil"
 )
 
-type Coeff struct {
-	Architecture  string  `csv:"architecture"`
-	CPUTime       float64 `csv:"cpu_time"`
-	CPUCycle      float64 `csv:"cpu_cycle"`
-	CPUInstr      float64 `csv:"cpu_instruction"`
-	MemoryUsage   float64 `csv:"memory_usage"`
-	CacheMisses   float64 `csv:"cache_misses"`
-	InterceptCore float64 `csv:"intercept_core"`
-	InterceptDram float64 `csv:"intercept_dram"`
-}
-
-type EnergyPrediction struct {
-	Architecture   string
-	CPUTime        float64
-	CPUCycle       float64
-	CPUInstr       float64
-	ResidentMemory float64
-	CacheMisses    float64
-}
-
-type CategoricalFeature struct {
-	Name   string  `json:"name"`
-	Weight float64 `json:"weight"`
-}
-
-type NormalizedNumericalFeature struct {
-	Mean     float64 `json:"mean"`
-	Variance float64 `json:"variance"`
-	Weight   float64 `json:"weight"`
-}
-
-type CoreModelServerCoeff struct {
-	AllCoreWeights struct {
-		BiasWeight           float64 `json:"Bias_Weight"`
-		CategoricalVariables struct {
-			CPUArchitecture []CategoricalFeature `json:"cpu_architecture"`
-		} `json:"Categorical_Variables"`
-		NumericalVariables struct {
-			CPUCycle NormalizedNumericalFeature `json:"cpu_cycles"`
-			CPUTime  NormalizedNumericalFeature `json:"cpu_time"`
-			CPUInstr NormalizedNumericalFeature `json:"cpu_instr"`
-		} `json:"Numerical_Variables"`
-	} `json:"core_All_Weights"`
-}
-
-type DramModelServerCoeff struct {
-	AllDramWeights struct {
-		BiasWeight           float64 `json:"Bias_Weight"`
-		CategoricalVariables struct {
-			CPUArchitecture []CategoricalFeature `json:"cpu_architecture"`
-		} `json:"Categorical_Variables"`
-		NumericalVariables struct {
-			CacheMisses    NormalizedNumericalFeature `json:"cache_misses"`
-			ResidentMemory NormalizedNumericalFeature `json:"container_memory_working_set_bytes"`
-		} `json:"Numerical_Variables"`
-	} `json:"dram_All_Weights"`
-}
-
-type LinearEnergyModelServerCoeff struct {
-	CoreModelServerCoeff
-	DramModelServerCoeff
-}
+type ModelOutputType int
+
+const (
+	AbsPower ModelOutputType = iota + 1
+	AbsModelWeight
+	AbsComponentPower
+	AbsComponentModelWeight
+	DynPower
+	DynModelWeight
+	DynComponentPower
+	DynComponentModelWeight
```

Comment:
I think we need to description about the `AbsPower` and the `DynPower`
Both a comment here and a small documentation